application/data/charts.py

```diff
@@ -14,26 +19,24 @@ def build(chart_object):
             builder = PanelLineChartObjectDataBuilder
 
         if builder:
-            return builder.build(chart_object)
+            return builder.build(dimension_chart, chart_object)
```

Comment:
This is a bit weird now as the chart lives inside `dimension_chart` anyway so we're effectively passing it in twice.  But like you say... `the coming months` should fix it :stuck_out_tongue:

Reply:
It is slightly weird, but at the moment necessary because of the Panel chart builders.